### PR TITLE
Refine homepage

### PR DESCRIPTION
**Pull request by tocean**
**Description**
Revise homepage 

**Major Revision**
- Add feature description: O1,O2,Training Example
- Move optimization level part to separate section
- Change to use figure rather than table for memory saving part

### CODE REVIEW

**Review by cp5555 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -131,16 +137,21 @@ Here here details of different MS-AMP optimization levels:
 
 We evaluated the training loss and validation performance of a typical model, Swin-Transformer, using both MS-AMP O2 and FP16 AMP. Our observations showed that the model trained with MS-AMP O2 mode achieved comparable performance to those trained using FP16 AMP. This demonstrates the effectiveness of the Mixed FP8 O2 mode in MS-AMP.
 
+Here is the result for Swin-Transformer tiny model:
```
Comment (wkcn) [inline README.md:140]: Swin-Transformer tiny -> Swin-T

https://github.com/microsoft/swin-transformer#main-results-on-imagenet-with-pretrained-models

File: README.md [hunk 2]
```diff
@@ -131,16 +137,21 @@ Here here details of different MS-AMP optimization levels:
...
+
 ### Memory
 
 MS-AMP preserves 32-bit accuracy while using only a fraction of the memory footprint on a range of tasks, including the DeiT model and Swin Transformer for ImageNet classification. For example, for a model with 1 billion parameters, MS-AMP with O2 mode can save approximately 12GB of GPU memory compared to FP16 AMP.
 
-| Model      | # Parameters(Billion) | Batch Size | FP16 AMP GPU Memory(MB) | MS-AMP GPU Memory(MB) |
-| -----------| --------------------- | ---------- | ----------------------  | --------------------- |
-| Swin       | 1.0                   | 16         | 34,322                  |  22,414               |
-| DeiT       | 1.2                   | 128        | 62,696                  |  48,328               |
+Here is the result for Swin-Transformer giant model and DeiT large model:
```
Comment (wkcn) [inline README.md:150]: 1. Swin-Transfomer giant -> Swin-giant-1B
Reference: SwinV2-giant-1B
https://github.com/microsoft/swin-transformer#updates

2. DeiT Large -> ViT-L/16
no DeiT Large mdel in DeiT family
We can call it ViT-L/16.

DeiT: https://github.com/facebookresearch/deit/blob/main/README_deit.md
ViT-L/16: https://arxiv.org/pdf/2010.11929.pdf

File: README.md [hunk 3]
```diff
@@ -131,16 +137,21 @@ Here here details of different MS-AMP optimization levels:
...
+
+![Image](./docs/assets/gpu-memory.png)
```
Comment (wkcn) [inline README.md:152]: Please note the unit of Y-axis (MB or GB).

Does MSAMP-FP8 represent Mixed FP8 O2?

**Review by wkcn — COMMENTED** [review]
Thanks Yuxiang for the refinement!

The model names should be corrected.

DeiT Family: DeiT-S (https://arxiv.org/pdf/2012.12877)
ViT Family: ViT-L/16 (https://arxiv.org/pdf/2010.11929.pdf)
Swin Transformer: Swin-T (https://arxiv.org/pdf/2103.14030)

File: README.md (hunk 2 above)
Comment (tocean) [reply README.md]: Change Swin-Transfomer giant to Swin-giant-1B and DeiT Large to ViT-L/16.

File: README.md [hunk 4]
```diff
@@ -4,6 +4,12 @@ MS-AMP is an automatic mixed precision package for deep learning developed by Mi
 
 Features:
 
+- Support O1 optimization: Apply FP8 to weights and weight gradients and support FP8 in communication.
+- Support O2 optimization: Support FP8 for two optimizizers(Adam and AdamW).
```
Comment (wkcn) [inline README.md:8]: two optimizizers(Adam and AdamW). -> two optimizers (Adam and AdamW).

File: README.md [hunk 5]
```diff
@@ -131,16 +137,21 @@ Here here details of different MS-AMP optimization levels:
...
+
 ### Memory
 
 MS-AMP preserves 32-bit accuracy while using only a fraction of the memory footprint on a range of tasks, including the DeiT model and Swin Transformer for ImageNet classification. For example, for a model with 1 billion parameters, MS-AMP with O2 mode can save approximately 12GB of GPU memory compared to FP16 AMP.
 
-| Model      | # Parameters(Billion) | Batch Size | FP16 AMP GPU Memory(MB) | MS-AMP GPU Memory(MB) |
-| -----------| --------------------- | ---------- | ----------------------  | --------------------- |
-| Swin       | 1.0                   | 16         | 34,322                  |  22,414               |
-| DeiT       | 1.2                   | 128        | 62,696                  |  48,328               |
+Here is the result for Swin-giant-1B model and ViT-L/16 model:
```
Comment (wkcn) [inline README.md:150]: Swin-giant-1B model and ViT-L/16 model -> Swin-1B and ViT-1.2B models.

File: README.md [hunk 6]
```diff
@@ -131,16 +137,21 @@ Here here details of different MS-AMP optimization levels:
...
+
+![Image](./docs/assets/gpu-memory.png)
+
+For detailed setting and results, please go to [MS-AMP-Example](https://github.com/Azure/MS-AMP-Examples).
```
Comment (wkcn) [inline README.md:154]: It is duplicated with L144.

File: README.md (hunk 5 above)
Comment (tocean) [reply README.md]: Fixed.

File: README.md (hunk 4 above)
Comment (tocean) [reply README.md]: Fixed.

**Review by wkcn — APPROVED** [review]
LGTM. Thanks Yuxiang!

File: README.md (hunk 6 above)
Comment (tocean) [reply README.md:152]: Fixed. Only keep one copy of it.
